### PR TITLE
Change admin room model

### PR DESCRIPTION
**Pull request by tritonJS826**

### CODE REVIEW

File: gf-webapp-admin/src/service/RoomService.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {RoomApi} from "../autogenerated/gf-server-api/apis/RoomApi";
+import {RoomsApi} from "../autogenerated/gf-server-api/apis/RoomsApi";
+import {Room} from "../model/Room/Room";
+import {NewRoom} from "../model/Room/NewRoom";
+import {Price} from "../model/Price";
+
+/**
+ * Temporary....... asdasdas asd akjsd alkjsd hajsd aljksdh lakjsd al
```
Comment (tritonJS826) [inline gf-webapp-admin/src/service/RoomService.ts:9]: )

File: gf-webapp-admin/src/utils/enumToArray.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * @param anyEnum - any enum
+ * @returns
```
Comment (tritonJS826) [inline gf-webapp-admin/src/utils/enumToArray.tsx:4]: add description

Comment (scorpigg) [reply gf-webapp-admin/src/utils/enumToArray.tsx:4]: done

File: gf-webapp-admin/src/service/RoomService.ts (hunk 1 above)
Comment (scorpigg) [reply gf-webapp-admin/src/service/RoomService.ts]: done

File: gf-webapp-admin/src/utils/enumToArray.tsx (hunk 2 above)
Comment (tritonJS826) [reply gf-webapp-admin/src/utils/enumToArray.tsx:4]: try to add "@descriptoin" on the line before param and describe "@return"

Comment (scorpigg) [reply gf-webapp-admin/src/utils/enumToArray.tsx]: done

**Review by tritonJS826 — COMMENTED** [review]
Good!

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**